packages/pigeon/platform_tests/test_plugin/macos/Classes/TestPlugin.swift

```diff
@@ -6,14 +6,34 @@ import Cocoa
 import FlutterMacOS
 
 /**
- * This plugin is currently a no-op since only unit tests have been set up.
- * In the future, this will register Pigeon APIs used in integration tests.
+ * This plugin handles the native side of the integration tests in
+ * example/integration_test/.
  */
-public class TestPlugin: NSObject, FlutterPlugin {
+public class TestPlugin: NSObject, FlutterPlugin, AllVoidHostApi, HostEverything {
   public static func register(with registrar: FlutterPluginRegistrar) {
+    let plugin = TestPlugin()
+    AllVoidHostApiSetup.setUp(binaryMessenger: registrar.messenger, api: plugin)
+    HostEverythingSetup.setUp(binaryMessenger: registrar.messenger, api: plugin)
   }
 
   public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
```

Comment:
Can you not pull this out?  It's a bit confusing to have.

Reply:
I forgot it was marked as `@optional` in the protocol; removed.